docs/api-docs/apps/guide/apps-13-publishing.md

```diff
@@ -76,19 +74,25 @@ Provide a helpful description with screenshots and a video to promote your app.
 | App Screenshot             | Screenshots of app UI inside BigCommerce                                                                                                  |
 | Alternate Logo             | Used if app featured in Marketplace carousel; should be 259 x 158px (or larger at ratio); dark background and light branding; no taglines |
 
+## App Supported Features
+
+Indicate whether your app supports multi-storefront functionality.
+
+![App Supported Features](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/devtools-msf.png "App Supported Features")
+
 ## Review submission
 
 Review the information added before submitting the app.
 
-![DevTools 04](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/devtools-tutorial-04.png  "DevTools 04")
+![Review](https://storage.googleapis.com/bigcommerce-production-dev-center/images/app-api-account/devtools-review.png "Review")
 
 ## Submit your app for approval
 
-Ensure all information is complete; test the app before submitting for review.
+Ensure all information is complete, then click **Submit for Review** to pay the review and listing fee and submit the app. Test the app before submitting to avoid pay additional review fees.

```

Comment:
```suggestion
Ensure all information is complete, then click **Submit for Review** to pay the review and listing fee and submit the app. Test the app before submitting it to avoid paying additional review fees.

```

Reply:
eek! thanks.